### PR TITLE
Update README.rst with set up of CT

### PR DESCRIPTION
**Pull request by NoyanAziz**
**Merge checklist:**
Check off if complete *or* not applicable:
- [x] Documentation updated (not only docstrings)
- [x] Fixup commits are squashed away
- [x] Unit tests added/updated
- [x] Manual testing instructions provided
- [x] Noted any: Concerns, dependencies, migration issues, deadlines, tickets

**Post-merge:**
- [ ] [Backported](https://openedx.atlassian.net/wiki/spaces/COMM/pages/2065367719/Making+a+pull+request+for+a+named+release) to latest and next named releases

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -80,6 +80,42 @@ Local
...
+
+  3. Go to the following URL to update the **waffle flags**:
+     http://localhost:8140/admin/waffle/flag/
+
+     Add the `transition_to_commercetools.redirect_to_commercetools_checkout` flag with the **Everyone** attribute set to **Yes**.
+
+  4. Similarly, go to:
+     http://localhost:18000/admin/waffle/flag/
+
+     Add the `commerce.transition_to_coordinator.checkout` flag with the **Everyone** attribute set to **Yes**.
```
Comment (aht007) [inline README.rst:101]: nit: We can add this flag as a sub bullet in point 3

File: README.rst [hunk 2]
```diff
@@ -80,6 +80,42 @@ Local
...
+
+
+Setup for running Refunds flow (**For refunds flow only**)
```
Comment (aht007) [inline README.rst:104]: This can be added as a sub section inside the `Setup with Commercetools` since it relates to commercetools development flow.

**Comment by aht007** [comment]
Please also sqush the commits locally and push again with correct commit name and type so that commitlint passes

File: README.rst [hunk 3]
```diff
@@ -80,6 +80,42 @@ Local
...
+==========================================================
+
+  Add the following inside your `'edx-platform/lms/envs/private.py'` file
+
+  .. code-block:: console
+
+    COMMERCE_COORDINATOR_REFUND_SOURCE_SYSTEMS = ('commercetools',)
+    COMMERCE_COORDINATOR_URL_ROOT = 'http://host.docker.internal:8140'
+
+  Go to the following URL to update the **waffle flags**:
```
Comment (aht007) [inline README.rst:114]: Parting thoughts:
Maybe we can list all of the waffle flags related to `commercetools` under a separate section....

File: README.rst [hunk 4]
```diff
@@ -80,6 +80,42 @@ Local
   # You should see JSON output indicating that two receivers were called, one successful, and one with exception/traceback information.
   # In the shell where the server is running you should see log output indicating that two test receivers were called with the sender argument "Something".
 
+Setup with Commercetools
```
Comment (shafqatfarhan) [inline README.rst:83]:
```suggestion
Local setup with Commercetools
```

File: README.rst (hunk 2 above)
Comment (shafqatfarhan) [inline README.rst:104]:
```suggestion
Local setup for running Refunds flow (**For refunds flow only**)
```

**Review by shafqatfarhan — APPROVED** [review]

**Review by NawfalAhmed — APPROVED** [review]

File: README.rst (hunk 1 above)
Comment (NoyanAziz) [reply README.rst:101]: Different URLs That is why kept them separate.

**Pull request merged**